front-end-admin/src/components/callout/CalloutInfo.vue

```diff
@@ -1,54 +1,86 @@
 <template>
   <div class="jumbotron">
     <div class="container">
-      <h2>{{callout.title}}</h2>
-      <!-- TODO: List out associated needs with a specific callout -->
-      <h2>Update Callout Information</h2>
-      <form v-on:submit.prevent="onFormSubmit">
-        <div class="form-group">
-          <label class="control-label col-sm-2">Title:</label>
-          <input v-model="callout.name" type="text" width="50" required>
+      <br>
+      <br>
+      <b-card title="Edit Callout">
+      <b-card :title="callout.name">
+        <b-form @submit="onFormSubmit" class="form-horizontal" >
```

Comment:
Make sure that the prevent default is added for the form so the page doesn't reload (unless this is what it should do)